README.md

```diff
@@ -6,6 +6,9 @@
 
 Blog website that shows a list of posts and empower readers to interact with them by adding comments and liking posts.
 
+![blog](https://user-images.githubusercontent.com/75126481/140084166-430a9c0c-55f3-4d9e-9c85-aebd85784bf0.png)
+
+
```

Comment:
**Changes Required**
- [ ] The README acts as your living documentation. Currently, the steps to successfully run the app is incomplete. Please let us update the README with all the steps required to run the app successfully on a local machine.